src/System.Diagnostics.FileVersionInfo/System.Diagnostics.FileVersionInfo.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Did we mean for this one to come into this PR? I'm not sure it makes a difference though